index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />	<link rel="stylesheet" href="index.css" type="text/css">
+	<title>Awesome Books Website</title>
+</head>
+<body>
+	<section id="container">
+		<header id="header">
+			<nav id="nav-bar">
+				<h4 class="logo">Awesome Books</h4>
+				<ul id="nav-items">
+					<li class="items" id="list">List</li>
+					<li class="items" id="add">Add</li>	
+					<li class="items" id="contact">Contact Us</li>
+				</ul>
+                </nav>
+				<p class="date"></p>
+		</header>
```

Comment:
# [ OPTIONAL CHANGES ] 
These changes won't stop your project from being approved but will help improve the look and feel a greater way 👍 
- [ ] It would be nice if you could do some more styling on the layouts of the elements highlighted in the screenshots below so that items have margins left and right and not seat at the left and right edge of the screen. This will improve the design looks
- [ ] Also, if you could add a cursor: pointer style to the nav links so we know we're hovering over a link to improve the user experience of your app 👍 

**See the screenshot below**
![image](https://user-images.githubusercontent.com/50941074/192321422-fdb46ca7-7339-48b4-81f8-a508d25d65f1.png)